src/tests/configs/nodes.json

```diff
@@ -2,37 +2,37 @@
   {
```

Comment:
it might be worth adding this to the gitignore since it just makes the PR's look ugly if someone accidentally commits it and its already in the repo, not sure if that would break anything
@Evanfeenstra what do you think?

Reply:
Definitely lets gitignore it. Im going to merge this for now tho